jenkins/shared-library/vars/createMultibranchJob.groovy

```diff
@@ -1,5 +1,21 @@
 // groovylint-disable-next-line MethodSize
-void call(Map jobConfiguration) {
+void call(Map jobConfiguration, Boolean manualTrigger = true) {
+	final String triggerText = manualTrigger ? '''
+				buildStrategies {
+					// By default, Jenkins will trigger builds as it detects changes on the source repository. We want
+					// to avoid that since we will trigger child pipelines on our own only when relevant.
+					skipInitialBuildOnFirstBranchIndexing()
+				}
+				strategy {
+					defaultBranchPropertyStrategy {
+						props {
+							noTriggerBranchProperty()
+						}
+					}
+				}
+	'''
```

Comment:
is this over indented ?

Reply:
yea, I was leaving the indent to match where it got inserted below.  but this is probably not needed since no one sees the final output. 😅